setup.py

```diff
@@ -2,5 +2,5 @@
 
 import os
 
-os.chdir('bindings/python')
-exec(open('setup.py').read())
+os.chdir("bindings/python")
+exec(open("setup.py").read())
```

Comment:
unrelated, but this turned out to be problematic btw. see https://github.com/arvidn/libtorrent/commit/e7e5805c56873ddfdcc09c407251185234caafbc#r46496265

Reply:
I think your fix there looks reasonable enough, but in reality I think `setup.py` executing a different file is a brittle approach. I wanted to experiment with moving all of `bindings/python/setup.py` into the root `setup.py`. I think it's less brittle but haven't experimented with this yet so I don't know the tradeoffs.